package.json

```diff
@@ -11,6 +11,7 @@
     "@material-ui/icons": "4.11.2",
     "@material-ui/lab": "4.0.0-alpha.57",
     "@material-ui/pickers": "3.2.7",
+    "amm-tools": "git+ssh://git@gitlab.cern.ch:7999/eis/amm/amm-tools.git#6de974466c4cfa74a5bec15b6e0d7a62bf6d8baf",
```

Comment:
amm-tools is not available publicly. Since the list of watchers is now present below the autocomplete, you can use the regular one. If you really want the AMMSingleAutocomplete, this dependency should only be included if the CERN profile is selected.